BuildMacOSUniversalBinary.py

```diff
@@ -314,6 +310,19 @@ def build(config):
     src_app1 = ARCHITECTURES[1]+"/Binaries/"
 
     recursive_merge_binaries(src_app0, src_app1, dst_app)
+
+    helpers_path = os.path.join(dst_app, "Dolphin.app", "Contents", "Helpers")
```

Comment:
Yeah, it might be easier to just use the existing directory. I'll have to double check that notarization still works properly, though.

(There really isn't any standard directory for embedded app bundles. Chromium embeds helper apps in ``Contents/Resources``, Instruments from Xcode embeds in `Contents/Helpers`, and Firefox embeds in 
`Contents/MacOS`...)